Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -899,7 +899,8 @@ void Element::make_html_uppercased_qualified_name()
 // https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task
 HTML::TaskID Element::queue_an_element_task(HTML::Task::Source source, Function<void()> steps)
 {
-    return queue_a_task(source, HTML::main_thread_event_loop(), document(), JS::create_heap_function(heap(), move(steps)));
+    auto& global_object = HTML::relevant_global_object(*this);
+    return HTML::queue_global_task(source, global_object, JS::create_heap_function(heap(), move(steps)));
```

Comment:
I'm not seeing the connection between this change and the test being added, and there's nothing in the commit message that explains what's going on.

Please at least explain the issue in the commit message so that we know why this change makes sense :)